ckanext/datagovuk/plugin.py

```diff
@@ -325,7 +325,7 @@ def get_helpers(self):
     ]
 
     def before_send(self, event, hint):
-        return None if [i for i in ['localhost', 'integration', 'staging'] if i in config.get('ckan.site_url')] or \
+        return None if [i for i in ['localhost', 'integration'] if i in config.get('ckan.site_url')] or \
```

Comment:
Same question here - why enable it in staging? I would assume you want to enable it in staging but _disable_ it in production, in order to try this out...?

Reply:
We also want errors from Staging, originally I disabled Staging errors first as they were more numerous due to data sync causing lots of data errors being thrown but have now mitigated that by ignoring data errors.